mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  return a * b * c;
+  
 }
```

Comment:
That's true, the `return` keyword and your operation must be in the same line. Or, another thing you can do is you can define a variable and and make it equal to `a * b * c`. After doing that, you can directly return the variable.